iot_input_oca/models/iot_device_input.py

```diff
@@ -14,7 +14,9 @@ class IotDeviceInput(models.Model):
     _order = "name"
 
     name = fields.Char(required=True)
-    device_id = fields.Many2one("iot.device", required=True, readonly=True)
+    device_id = fields.Many2one(
+        "iot.device", required=True, readonly=True, auto_join=True
```

Comment:
We are setting autojoin in order to make `parse_args` domain faster. At the end, if you can see an input, you should be able to see the device